docs/3_guides/0_submitting_proofs.md

```diff
@@ -141,10 +145,10 @@ aligned submit \
 --proving_system SP1 \
 --proof ./scripts/test_files/sp1/sp1_fibonacci.proof \
 --vm_program ./scripts/test_files/sp1/sp1_fibonacci.elf \
---batcher_url wss://batcher.alignedlayer.com \
+--batcher_url wss://mainnet.batcher.alignedlayer.com \
```

Comment:
Use holesky on the predefined examples. It could lead to users sending funds in mainnet by error